From 35b84ef2fa6a0b24a9b37dc3744afc6e3b7840ed Mon Sep 17 00:00:00 2001
From: Sultan Alsawaf <sultan@kerneltoast.com>
Date: Sat, 26 Dec 2020 10:55:58 -0800
Subject: [PATCH 142/175] drivers: gpu: msm: Force GPU idle timeout to 58 ms

This matches the ULPS timeout used in msm_drm, which is defined as 58 ms
by IDLE_POWERCOLLAPSE_DURATION. No need to keep the GPU running after
the display enters a low-power state.

This also prevents userspace from changing the GPU idle timeout.

Signed-off-by: Sultan Alsawaf <sultan@kerneltoast.com>
---
 drivers/gpu/msm/kgsl_pwrctrl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/msm/kgsl_pwrctrl.c b/drivers/gpu/msm/kgsl_pwrctrl.c
index efc0e77da52a..81e86b388208 100644
--- a/drivers/gpu/msm/kgsl_pwrctrl.c
+++ b/drivers/gpu/msm/kgsl_pwrctrl.c
@@ -1063,6 +1063,8 @@ static ssize_t __timer_store(struct device *dev, struct device_attribute *attr,
 	struct kgsl_device *device = dev_get_drvdata(dev);
 	int ret;
 
+	return count;
+
 	ret = kgsl_sysfs_store(buf, &val);
 	if (ret)
 		return ret;
-- 
2.48.1

